dealroom_firestore_connector/__init__.py

```diff
@@ -407,6 +407,31 @@ def _get_final_url_and_dealroom_id(
     return final_url, dealroom_id
 
 
+def check_for_deleted_profiles(
+    key_found, history_refs, dealroom_id, count_history_refs
+):
+    # https://dealroom.atlassian.net/browse/DS2-154
+    document_matches_by_final_url = key_found == "final_url"
```

Comment:
I would put this check outside of this function: would you evaluate the condition to call a function outside the function, or would you evaluate if the function needs to do smthg after you've called it?

Reply:
Evaluating a condition is less complex since it's just one line in machine code, while calling a function requires at least two. So I'll update it

Reply:
Updated in [7a914f9](https://github.com/dealroom/data-firestore-connector/pull/28/commits/7a914f91e82b68cb2a41ee63d610d3b9893b00a9)

Reply:
ok, but really what's important here is readability: evaluating the condition inside the function forces the reader to read code that is nested more than it should be. Evaluating the condition outside the function also improves readability, since you already are calling the funct conditionally